admin/app/components/team/add-member.js

```diff
@@ -0,0 +1,75 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import isEmailValid from '../../utils/email-validator';
+
+export default class AddMember extends Component {
+  @service notifications;
+  @service store;
+
+  @tracked email = '';
+  @tracked role = 'SUPER_ADMIN';
+  @tracked inviteErrorRaised;
+
+  @action
+  async inviteMember(event) {
+    event.preventDefault();
+
+    if (!this._isEmailValid()) {
+      return;
+    }
+
+    const members = this.store.peekAll('admin-member');
+    if (members.find((member) => member.email === this.email)) {
+      return this.notifications.error('Cet agent a déjà accès');
+    }
+
+    try {
+      const adminMember = this.store.createRecord('admin-member', { email: this.email, role: this.role });
+      await adminMember.save();
+      this.email = '';
+      this.role = 'SUPER_ADMIN';
+      this.notifications.success(`L'agent a dorénavant accès`);
```

Comment:
Il manque un bout de phrase ^^
```suggestion
      this.notifications.success(`L'agent a dorénavant accès à Pix Admin avec le rôle ???`);
```

Reply:
Un test unitaire des messages d'erreur et de succès pourront couvrir ça ? :)

Reply:
On ne trouve pas pertinant de remettre le role dans la notification parce qu'il apparait dans le tableau avec le role, c'est un peu overkill, mais on peut checker avec Emmy lundi pour voir ce qu'elle prefere